### ISSUE TITLE
[BUG]

### ISSUE THREAD

**Issue by KahunaLadd** [Chia-Network/chia-blockchain#5795]
Can't connect to wallet, app locked up.

I had tried to run 4 pots in parallel, but ran out of harddrive space "NOOB". Nothing happened all night, they were stuck at their %'s. The Temporary hard drive was completely full. I could not stop the plots, so I killed everything with task manager. I then deleted everything in the temp folder. When I restarted chia, it was just locked looking for wallet. I then uninstalled chia, then reinstalled. Still stuck looking for wallet. I then uninstalled again, did a registry search for anything related to chia, deleted the app data, reinstalled, then ran again. Still hangs up looking for wallet.
Any help would be appreciated.

**Comment by mariano54** [comment]
Duplicate of https://github.com/Chia-Network/chia-blockchain/issues/6993, please check and discuss in that issue. We are consolidating all wallet connection and sync issues there.
